src/data/fintype/basic.lean

```diff
@@ -425,6 +425,14 @@ set.ext $ λ _, mem_to_finset
   s.to_finset = t.to_finset ↔ s = t :=
 ⟨λ h, by rw [← s.coe_to_finset, h, t.coe_to_finset], λ h, by simp [h]; congr⟩
 
+@[simp] theorem to_finset.mono {s t : set α} [fintype s] [fintype t] :
+  s.to_finset ≤ t.to_finset ↔ s ≤ t :=
+by simp [finset.subset_iff, set.subset_def]
```

Comment:
I assume this doesn't work with `monotone` due to the fintype arguments?

Can you add a `strict_mono` version too, and tag both with `@[mono]`?

Reply:
The linter wants
```suggestion
@[simp] theorem to_finset.mono {s t : set α} [fintype s] [fintype t] :
  s.to_finset ⊆ t.to_finset ↔ s ⊆ t :=
by simp [finset.subset_iff, set.subset_def]
```